### ISSUE TITLE
Port to Distributions.jl?

### ISSUE THREAD

**Issue by azev77** [bdeonovic/GeneralizedLambdaDistribution.jl#1]
Hi @bdeonovic and thanks for this awesome package!
Is there a change you can port it to Distributions.jl?

**Comment by bdeonovic** [comment]
I can look into it, its pretty old code now that I used awhile ago for a project so I might have to go through it.

**Comment by azev77** [comment]
@bdeonovic that would be awesome!
Btw, I just realized you have a goldmine of very flexible distributions: 
`GLD{T}(lambda1::T, lambda2::T, lambda3::T, lambda4::T)`
Pearson Types I-VII
`GK(A::Real, B::Real, g::Real, k::Real, c::Real)`
wow!

I bet 100 name-brand distributions can be written as a special case of these three: https://threeplusone.com/pubs/FieldGuide.pdf

**Comment by bdeonovic** [comment]
Yes, the Pearson one is particularly useful. I used these packages in a project where I was trying to approximate a difficult to evaluate distribution.
